celery/backends/filesystem.py

```diff
@@ -53,10 +53,9 @@ def __init__(self, url=None, open=open, unlink=os.unlink, sep=os.sep,
         # Lets verify that we've everything setup right
         self._do_directory_test(b'.fs-backend-' + uuid().encode(encoding))
 
-    def __reduce__(self, args=(), kwargs={}):
-        kwargs.update(
-            dict(url=self.url))
-        return super().__reduce__(args, kwargs)
+    def __reduce__(self, args=(), kwargs=None):
+        kwargs = {} if not kwargs else kwargs
+        return super().__reduce__(args, dict(kwargs, url=self.url))
```

Comment:
```suggestion
        return super().__reduce__(args, {**kwargs, 'url': self.url})
```

With this change it works. Is this preferable or should I go with the first option? I usually prefer avoiding a string literal if possible.